src/app/src/main/kotlin/org/imperial/mrc/hint/clients/FuelClient.kt

```diff
@@ -42,7 +42,13 @@ abstract class FuelClient(protected val baseUrl: String)
 
     protected fun postJson(url: String, json: String): ResponseEntity<String>
     {
-        return "$baseUrl/$url".httpPost()
+        val urlPath = when
+        {
+            baseUrl.isNullOrEmpty() -> url
```

Comment:
As above, this is the wrong way round... `url` can be optional, but not `baseUrl`. Otherwise we're inconsistent with the other, existing, clients.